crates/swc_ecma_ast/src/module.rs

```diff
@@ -119,11 +119,14 @@ pub enum ModuleItem {
     #[tag("TsImportEqualsDeclaration")]
     #[tag("TsExportAssignment")]
     #[tag("TsNamespaceExportDeclaration")]
-    ModuleDecl(ModuleDecl),
+    ModuleDecl(Box<ModuleDecl>),
     #[tag("*")]
-    Stmt(Stmt),
+    Stmt(Box<Stmt>),
```

Comment:
Yeah I profiled it using the minifier

Reply:
The performance gain was about 10% (8% ~ 15%), on total time

Reply:
I'll run it one more time

Reply:
Hmm... I can't reproduce it.

Reply:
If there are gains, I bet most of it is from the change on `BlockStmt` because that will be more prevalent than top level module items (generally, most files will have a low number of module items). It seems most of the nightly feature is used to support the boxing here though.

Reply:
Sounds reasonable. I'll wait for the profiling result as I requested one to someone